models/issue_milestone.go

```diff
@@ -178,10 +178,11 @@ func (milestones MilestoneList) getMilestoneIDs() []int64 {
 	return ids
 }
 
-// GetMilestonesByRepoID returns all milestones of a repository.
+// GetMilestonesByRepoID returns all opened milestones of a repository.
 func GetMilestonesByRepoID(repoID int64) (MilestoneList, error) {
 	miles := make([]*Milestone, 0, 10)
-	return miles, x.Where("repo_id = ?", repoID).Asc("deadline_unix").Find(&miles)
+	return miles, x.Where("repo_id = ? AND is_closed = ?", repoID, 0).
```

Comment:
it should be not `0` but `false`

Reply:
Oh, `0` was in the db when I looked... Would fix